Builds failing with subset violation

Official builds are failing on a number of architectures with the following:

> /__w/1/s/eng/liveBuilds.targets(89,5): error : The CoreCLR artifacts path does not exist '/datadisks/disk1/workspace/_work/1/s/artifacts/transport/coreclr'. The CoreCLR subset category must be built before building this project. [/__w/1/s/src/libraries/System.Buffers/src/System.Buffers.csproj]

There have been similar bug reports in the past but there has never been a definitive repro. This has failed at least two builds in a row hence appears fairly definitive:

- https://dev.azure.com/dnceng/internal/_build/results?buildId=491110
- https://dev.azure.com/dnceng/internal/_build/results?buildId=491154

A rebase cleared this issue, but I was hitting a new error on the Installer builds that also seems to be CI related: https://dev.azure.com/dnceng/public/_build/results?buildId=493235&view=logs&j=edc799a8-89b7-5e0a-b9c0-c146c0eff005&t=a0500b23-24fd-5809-e193-9d1182e3b95d&l=671

```
##[error]/root/runtime/src/installer/pkg/projects/netcoreapp/sfx/Microsoft.NETCore.App.SharedFx.sfxproj(0,0): error NU1102: (NETCORE_ENGINEERING_TELEMETRY=Build) Unable to find package Microsoft.NETCore.Platforms with version (>= 5.0.0-ci.20073.1)
```

I'm pretty sure that's a new/different issue than this one though.

--- a/eng/pipelines/libraries/base-job.yml
+++ b/eng/pipelines/libraries/base-job.yml
@@ -90,10 +90,7 @@ jobs:
         - ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
           - _runtimeDownloadPath: '$(Build.SourcesDirectory)/artifacts/transport/${{ parameters.runtimeFlavor }}'
           - _runtimeArtifactName: '$(runtimeFlavorName)Product_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.liveRuntimeBuildConfig }}'
-          - ${{ if ne(parameters.container, '') }}:
-            - _runtimeArtifactsPathArg: '/p:RuntimeArtifactsPath=${_RUNTIMEDOWNLOADPATH}'
-          - ${{ if eq(parameters.container, '') }}:
-            - _runtimeArtifactsPathArg: ' /p:RuntimeArtifactsPath=$(_runtimeDownloadPath)'
+          - _runtimeArtifactsPathArg: ' /p:RuntimeArtifactsPath=$(_runtimeDownloadPath)'
 
           # WebAssembly uses linux implementation detail
           - ${{ if eq(parameters.osGroup, 'WebAssembly') }}:

